src/components/MAPI/workernodes/guides/ListNodePoolsGuide.tsx

```diff
@@ -54,7 +54,7 @@ const ListNodePoolsGuide: React.FC<IListNodePoolsGuideProps> = ({
       <CLIGuideStepList>
         <LoginGuideStep />
         <CLIGuideStep
-          title='2. List node pools.'
+          title='2. List node pools'
```

Comment:
Moved this guide to the correct folder and removed the period at the end of step 2 to align with the other guides